deployment/deploy-tkg.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Postfacto, a free, open-source and self-hosted retro tool aimed at helping
+# remote teams.
+#
+# Copyright (C) 2016 - Present Pivotal Software, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+#
+# it under the terms of the GNU Affero General Public License as
+#
+# published by the Free Software Foundation, either version 3 of the
+#
+# License, or (at your option) any later version.
+#
+#
+#
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+#
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#
+# GNU Affero General Public License for more details.
+#
+#
+#
+# You should have received a copy of the GNU Affero General Public License
+#
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+set -euo pipefail
+
+if [ $# -lt 1 ]; then
+  echo "usage: ./deploy.sh <app-name> [<kubeconfig-path>]"
```

Comment:
Do we actually use the `kubeconfig-path` anywhere?

Reply:
Removed as unused. We ended up remove it's usage to simplify the script.